test/user/model.js

```diff
@@ -24,12 +24,12 @@ describe('<Unit Test>', function() {
         });
 
         describe('Method Save', function() {
```

Comment:
@seyi-adeleke This make sense. Kindly approve we can get this sorted, out of the way. we'll create another story to fix the test.